index.html

```diff
@@ -20,71 +20,223 @@
-          I can help you build a product , feature or website Look through some
-          of my work and experience! If you like what you see and have a project
-          you need coded, don’t hestiate to contact me.
-        </p>
-      </div>
-      <hr class="horizontal" />
-      <div class="qualifications">
+      <section class="jumbotron">
         <div class="white-background"></div>
-        <div>
-          <h4 class="head-color text-normal align-left roboto">Language</h4>
-          <ul>
-            <li class="text-400 align-left li-color roboto">JavaScript</li>
-            <li class="text-400 align-left li-color roboto">Ruby</li>
-            <li class="text-400 align-left li-color roboto">Html</li>
-            <li class="text-400 align-left li-color roboto">Css</li>
-          </ul>
+        <div class="my-title">
+          <h1 class="head-color align-center roboto wrap">William Raga</h1>
+          <p class="text-normal margin align-center text-color roboto wrap">
+            Software Developer
+          </p>
         </div>
         <div>
-          <h4 class="head-color text-normal roboto">Frameworks</h4>
-          <ul>
-            <li class="text-400 align-left li-color roboto">Bootstrap</li>
-            <li class="text-400 roboto align-left li-color">Ruby on rails</li>
-            <li class="text-400 roboto align-left li-color">RSPec</li>
-            <li class="text-400 roboto align-left li-color">CapyBara</li>
-            <li class="text-400 roboto align-left li-color">Selenium</li>
-          </ul>
+          <p class="text-400 align-center description roboto">
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
         </div>
-        <div>
-          <h4 class="head-color roboto text-normal align-left">Skills</h4>
-          <ul>
-            <li class="text-400 roboto align-left li-color">Codekit</li>
-            <li class="text-400 roboto align-left li-color">GitHub</li>
-            <li class="text-400 roboto align-left li-color">Codepen</li>
-            <li class="text-400 roboto align-left li-color">GitLab</li>
-            <li class="text-400 roboto align-left li-color">Terminal</li>
-          </ul>
+        <hr class="horizontal" />
+        <div class="qualifications">
+          <div class="white-background"></div>
+          <div>
+            <h4 class="head-color text-normal align-left roboto">Language</h4>
+            <ul>
+              <li class="text-400 align-left li-color roboto">JavaScript</li>
+              <li class="text-400 align-left li-color roboto">Ruby</li>
+              <li class="text-400 align-left li-color roboto">Html</li>
+              <li class="text-400 align-left li-color roboto">Css</li>
+            </ul>
+          </div>
+          <div>
```

Comment:
- [ ] Kindly encapsulate this entire section in a `ul` and then each of Languages, Frameworks, and Skills can be a `li` in itself. Since this is semantically a list as well, in addition to the list of its content; Putting them as a list will improve the semantic structure of the website.